### PR TITLE
Created issue templates

### PR DESCRIPTION
**Pull request by ArtBlue**
I started off the bug report and new feature issue templates based on where we landed on the checklist. I simply referred to the Wiki page for reference to the UX guidelines.

I don't know how the "Update feature request" will differ from "New feature request" so I didn't touch that yet.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-feature-request.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Add a link to the Figma page for the feature/module.
+
+**Prerequisites**
+
+- [ ] Successful design review with Design System Team.
+- [ ] Supports dark mode.
+- [ ] Supports responsive design.
+- [ ] Tokens have been provided (or already exist)
+- [ ] Successful accessibility review with a11y core team
+- [ ] Successful UX review with eBayUI Team (See UX Guidelines on Wiki)
```
Comment (agliga) [inline .github/ISSUE_TEMPLATE/new-feature-request.md:29]: I thought we left this off that the UX review was going to be the same as the design review with us?
I would change this with `successful review with ebayUI team`. Or even `reviewed with ebayUI team`. We can bring up the UX expectations there with them when we review.

File: .github/ISSUE_TEMPLATE/new-feature-request.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+**Figma Link**
+Add a link to the Figma page for the feature/module.
+
+**Prerequisites**
+
+- [ ] Successful design review with Design System Team.
+- [ ] Supports dark mode.
+- [ ] Supports responsive design.
+- [ ] Tokens have been provided (or already exist)
+- [ ] Successful accessibility review with a11y core team
```
Comment (agliga) [inline .github/ISSUE_TEMPLATE/new-feature-request.md:28]: I would simplify the langue like, `passed accessibility review` or `Reviewed with a11y team`

File: .github/ISSUE_TEMPLATE/new-feature-request.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+name: New Feature request
+about: Request a new feature for Skin
+title: "[NEW FEATURE]"
+labels: 'type: feature'
+assignees: ''
+
+---
+
+**Provide a desired name for the feature or module**
+A clear and concise name.
```
Comment (agliga) [inline .github/ISSUE_TEMPLATE/new-feature-request.md:11]: This is usually in the title, I don't think we need an extra callout.

File: .github/ISSUE_TEMPLATE/new-feature-request.md (hunk 1 above)
Comment (ianmcburnie) [inline .github/ISSUE_TEMPLATE/new-feature-request.md:29]:
```suggestion
- [ ] Successful frontend review with eBayUI Team
```

File: .github/ISSUE_TEMPLATE/new-feature-request.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+**Screenshots**
+Add screenshots of the feature/module.
+
+**Figma Link**
+Add a link to the Figma page for the feature/module.
+
+**Prerequisites**
+
+- [ ] Successful design review with Design System Team.
```
[the remark this answers is not in the document]
Comment (ArtBlue) [reply .github/ISSUE_TEMPLATE/new-feature-request.md:21]: I don't know what this was either. Maybe @ianmcburnie  knows.

**Comment by ArtBlue** [comment]
@ianmcburnie , @agliga , I updated the PR. There's one outstanding comment from Andrew that I don't know the answer to.

File: .github/ISSUE_TEMPLATE/new-feature-request.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ianmcburnie) [reply .github/ISSUE_TEMPLATE/new-feature-request.md:21]: It's if the feature requested has been signed off by design system team.

File: .github/ISSUE_TEMPLATE/new-feature-request.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+- [ ] Supports dark mode.
+- [ ] Supports responsive design.
+- [ ] Tokens have been provided (or already exist)
+- [ ] Passed accessibility review
```
Comment (ianmcburnie) [inline .github/ISSUE_TEMPLATE/new-feature-request.md:25]:
```suggestion
- [ ] Successful accessibility review with Core A11Y Team.
```

Comment (ianmcburnie) [reply .github/ISSUE_TEMPLATE/new-feature-request.md:25]: I'm not sure why we went back on the wording here, but I want to make it completely unambiguous that every new feature going into ebayui needs core a11y team review.

Comment (ArtBlue) [reply .github/ISSUE_TEMPLATE/new-feature-request.md:25]: That was @agliga 's suggestion.

Comment (agliga) [reply .github/ISSUE_TEMPLATE/new-feature-request.md:25]: Just wanted to simplify the wording. 
But we should do it across the board if we do it

Comment (ianmcburnie) [reply .github/ISSUE_TEMPLATE/new-feature-request.md:25]: Let's keep it explicit. Also, we may want a footnote to explain that "successful review" means those teams give their blessing for it to proceed to engineering.

Comment (ArtBlue) [reply .github/ISSUE_TEMPLATE/new-feature-request.md]: Done.

File: .github/ISSUE_TEMPLATE/new-feature-request.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+**Prerequisites**
+
+- [ ] Successful design review with Design System Team.
+- [ ] Supports dark mode.
+- [ ] Supports responsive design.
+- [ ] Tokens have been provided (or already exist).
+- [ ] Successful accessibility review with Core A11Y Team.
+- [ ] Successful frontend review with eBayUI Team.
+
+Note: A "successful review" means those teams give their blessing for it to proceed to engineering.
```
Comment (LuLaValva) [inline .github/ISSUE_TEMPLATE/new-feature-request.md:28]: Maybe put this in a comment too? I'm not sure if we need to see this message at the end of every issue

**Review by LuLaValva — APPROVED** [review]
Thanks for making the changes @ArtBlue, this looks great! One suggestion, but if you disagree I think it should be good to merge now, and we can do a follow up PR where we convert to `.yml` for issue forms.

File: .github/ISSUE_TEMPLATE/new-feature-request.md (hunk 6 above)
Comment (LuLaValva) [reply .github/ISSUE_TEMPLATE/new-feature-request.md:28]: Actually, I think it might be cool to put this in a note card

```
> **Note**
> A "successful review" means those teams give their blessing for it to proceed to engineering.
```

> **Note**
> A "successful review" means those teams give their blessing for it to proceed to engineering.

Comment (ianmcburnie) [reply .github/ISSUE_TEMPLATE/new-feature-request.md:28]: You can use GitHub's suggestion feature to make this easy for PR owner to commit.

```suggestion
> **Note**
> A "successful review" means those teams give their blessing for it to proceed to engineering.
```

**Comment by ArtBlue** [comment]
> Thanks for making the changes @ArtBlue, this looks great! One suggestion, but if you disagree I think it should be good to merge now, and we can do a follow up PR where we convert to `.yml` for issue forms.

@LuLaValva , I thought we were cool iterating on this in the future because we needed more exploration of forms (see details in thread) to make sure they were going to work for us. 

Alright, I give up. 😄  Tomorrow I'll create a separate branch and get a PR ready for this using forms. We can get that merged in and test it out to see if what's supported works for us out of the box. If so, we'll use that. If not, we'll revert that and use this instead.

*If we can't nail this down tomorrow merge one of these solutions, I think I'm going to put this on hold. I really need to focus on input chips if we're going to have a chance to release that in this upcoming release.*

**Comment by ArtBlue** [comment]
Closing this PR in lieu of using issue forms.
